specs/specs.bs

```diff
@@ -963,7 +962,7 @@ Also:
   - the value of `"values"` is a hierarchy of arrays with integers. The depth depends on the Geometry object: for `MultiPoint` and `MultiLineString` this is a simple array of integers; for any other geometry type it is two less than the array `"boundaries"`. An integer refers to the index in the `"surfaces"` array of the same geometry, and it is 0-based. If one surface has no semantics, a value of `null` must be used.
 
 <div class="note">
-  For legacy reasons, we use `"surfaces"` to name the array of Semantic Object. Nevertheless, this property is used for points and linestrings of `MultiPoints` and `MultiLineStrings`, as well.
+  For legacy reasons, we use `"surfaces"` to name the array of Semantic Objects. Nevertheless, this member is used for points and linestrings of `MultiPoints` and `MultiLineStrings`, as well.
```

Comment:
First we had semantics only for surfaces, but then someone (@liberostelios) pointed out that lines or points could have semantics too.

So it's possible to attach them, but the member is called "surfaces", so we added a warning.

Maybe not necessary to have that warning, since indeed can only but confuse people?

Reply:
I will leave it as is for now.